scripts/update-tasks.sh

```diff
@@ -189,6 +202,18 @@ main() {
     "s2i"  "image: quay.io/buildah"  \
```

Comment:
Can we remove this? Does it work then? It can come from the array, OPENSHIFT_CATALOG_TASKS also. Except buildah task which has default, other can from that array.

Reply:
s2i is in `TEKTON_CATALOG_TASKS`, not `OPENSHIFT_CATALOG_TASKS`

Reply:
OK.